devtools/common/src/main/java/io/quarkus/cli/commands/file/GradleBuildFile.java

```diff
@@ -55,9 +43,26 @@ public void close() throws IOException {
 
     @Override
     public void completeFile(String groupId, String artifactId, String version) throws IOException {
+        init();
         completeSettingsContent(artifactId);
         completeBuildContent(groupId, version);
-        completeProperties(artifactId);
+        completeProperties();
+    }
+
+    // this is only visible for testing purposes
```

Comment:
did you forget to commit some tests ? :)

Reply:
Nope, I forgot to remove this comment 🤪. I'm out for a short while, will update shortly

Reply:
Fixed